### DISCUSSION TITLE
Why Typst doesn't support arc

### DISCUSSION THREAD

**Discussion by mojtaba82** [typst/typst#3910] [General]
Typst not support arc and There exist no easy way for draw arc in Typst. CeTZ  for draw arc use path element and by bezier curve approximate it.

**Comment by Enivex** [comment]
I do not believe the PDF standard has arcs. They'll always be approximated by Bezier curves. The error should be small enough that it makes no difference in practice.

**Reply by mojtaba82** [reply]
I didn't know that. Thanks.
